src/desktop/apps/experimental-app-shell/client.tsx

```diff
@@ -12,6 +12,7 @@ const mediator = require("desktop/lib/mediator.coffee")
 buildClientApp({
   routes: getAppRoutes(),
   context: {
+    EXPERIMENTAL_APP_SHELL: sd.CLIENT_NAVIGATION_V3 === "experiment",
```

Comment:
same as https://github.com/artsy/force/pull/5138/files#r385207609 , this is always `true` right?